src/api/app/policies/workflow_run_policy.rb

```diff
@@ -1,9 +1,11 @@
 class WorkflowRunPolicy < ApplicationPolicy
-  class Scope
+  class Scope < Scope
     def initialize(user, scope, opts = {})
       @user = user
       @scope = scope
       @opts = opts
+
+      #super(user, scope)
```

Comment:
Hm, what's this about?

Reply:
Commenting this line was not on purpose. Why changing this policy now, maybe @danidoni can explain why this happend?

Reply:
I don't remember changing this line, so probably was an accident